### PR TITLE
Update Issue Templates to allow Feature Selection

### PR DESCRIPTION
**Pull request by brian-avery**
This updates the issue templates such that a "feature" section will be added to new templates. Checking features will apply the proper labels.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ about: Report a bug to help us improve Istio
 [ ] User Experience
 [ ] Developer Infrastructure
 
+**Affected features (please put an X in all that apply)**
+
+[ ] Multi Cluster
+[ ] Virtual Machine
+[ ] Multi Control Plane
```
Comment (sdake) [inline .github/ISSUE_TEMPLATE/bug_report.md]: The problem I expect with this template is there will be confusion about features not listed, or we will end up mirroring the [feature page](https://istio.io/about/feature-stages/).

We can reasonably expect any feature that is alpha to be more likely to be selected from this list. I'd suggest adding some language such as "Alpha Affected features ..."

From the alpha list, there remains this list:
- Operator
- CNI Plugin
- IPv6

Comment (brian-avery) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I'm hesitant to use the phrase  "alpha features" here. These are more overarching features that span multiple workgroups and I think issues may continue to get those labels even after they are no longer alpha. We're discussing in the workgroup leads meetings what other features to add.

Comment (linsun) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I assume this is only for cross WG features?  if so, I'm good with this.

Comment (brian-avery) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yep. This is for cross work-group features. Thanks.

**Pull request merged**